### DISCUSSION TITLE
Missing Link between Data and Schemas to Ontologies

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#37] [General]
There is a lack of ontologies for spatial data to help integrate INSPIRE data with non-INSPIRE data and to be able to establish ontological rules for the data. These ontologies would also help in searching the data, allowing for more tailored searches.

**Comment by thorsten-reitz** [comment]
Thuis has been true already 15 years ago when the original research for HUMBOLDT was conducted. At the time, we found that only 2% of dasta sets directly re-use concepts that are well-defined through an ontology.

WE have thus in the past used lifting, i.e. transforming other conmceptual data models into de facto ontologies. This was also applied by the taxonomy tools created by Minerva Intelligence (see here: https://italy.minervageo.com/inspire-data-challenge/).

**Reply by thorsten-reitz** [reply]
I think that https://github.com/gopeg/issues-barriers/issues/8 is a duplicate of this to some degree.

**Reply by AndersOstmanNovogit** [reply]
Now #46.

**Comment by AndersOstmanNovogit** [comment]
This issue have not yet been entered into D4.3, because I need some clarification about the actual issue being observed, not the hypothetical issues I can invent myself.

I see some different type of problems here. Which problems have been identified in the use cases and how did you solve them.

My different interpretations are as follows
1. The problem is that concepts being used in the data set/model are not properly defined or defined in a standardized way. This creates an uncertainty in how to process the data properly. The usage of standards such as SKOS and/or OWL is encouraged.
2. If the concepts being used are not defined in a well known ontology / vocabulary, a user do not know which concepts to use when searching for the data. A wider usage of domain ontologies / vocabularies is encouraged.
3. Many conceptual schemas of today are described by UNL (class) diagrams, which are less suitable for automated processing. Uplifting such conceptual models to de facto ontologies (OWL) would allow for better integration of concepts by machines.
